telemetry/definitions/commonDefinitions.json

```diff
@@ -2609,50 +2549,66 @@
             ],
             "passive": true
         },
+
         {
-            "name": "awsq_serviceInvocation",
-            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
-            "metadata": [
-                {
-                    "type": "awsqAssignClickCount"
-                },
-                {
-                    "type": "awsqGenerateCodeIterationCount"
-                },
-                {
-                    "type": "awsqGenerateCodeClickCount"
-                },
-                {
-                    "type": "awsqApproachIterationCount"
-                },
-                {
-                    "type": "awsqIsRegeneratedCode"
-                },
-                {
-                    "type": "awsqFilesReviewedCount"
-                },
-                {
-                    "type": "awsqIsAcceptedCodeChanges"
-                },
-                {
-                    "type": "awsqRepositorySize"
-                },
-                {
-                    "type": "awsqFilesChangedCount"
-                },
-                {
-                    "type": "awsqThumbsUpCount"
-                },
-                {
-                    "type": "awsqThumbsDownCount"
-                },
-                {
-                    "type": "awsqIsApproachAccepted"
-                },
-                {
-                    "type": "awsqServiceErrorsCount"
-                }
-            ]
+            "name": "awsq_assignCommand",
+            "description": "Number of times the user have triggered /assign ",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_codeReGeneration",
+            "description": "Number of times the user invoked agent to re-engage in Code Gen.",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_codeGenerateClick",
+            "description": "Number of times the user has clicked on Generate Code",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_approach",
+            "description": "Number of time the user received an Approach.",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_isRegeneratedCode",
+            "description": "User has re-generated code as a part of code generation portion",
+            "metadata": [{ "type": "enabled" }]
+        },
+        {
+            "name": "awsq_filesReviewed",
+            "description": "How many files the user reviewed files after Code Generation phase.",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_isAcceptedCodeChanges",
+            "description": "User applied code changes generated for the task.",
+            "metadata": [{ "type": "enabled" }] 
+        },
+        {
+            "name": "awsq_repo",
+            "description": "The size of the input repository",
+            "metadata": [{ "type": "awsqRepositorySize" }] 
```

Comment:
this metric could be `"unit": "Count"`, then `awsqRepositorySize` is not needed.

Alternatively, please consider naming the type as `filesize` or `directorysize` so that it can be re-used in other metrics.